content/300-guides/125-development-environment/050-environment-variables/index.mdx

```diff
@@ -130,3 +130,13 @@ import * as dotenv from 'dotenv'
 dotenv.config() // Load the environment variables
 console.log(`The connection URL is ${process.env.DATABASE_URL}`)
 ```
+
+If you are using environment variables in a custom file, you can add it to your code seperately using `dotenv`. If you need variable expansion across environment files, you will have to use [`dotenv-expand`](https://github.com/motdotla/dotenv-expand):
+
+```ts
+const dotenvExpand = require('dotenv-expand')
+
+var envFile = path.resolve(join(__dirname, "myenv.env"))
+var mySqlEnv = dotenv.config({path: envFile})
+dotenvExpand.expand(mySqlEnv)
+ ```
```

Comment:
````suggestion
If you are using a custom file name for your environment variables, you can configure `dotenv` to use that filename:

```ts
import * as dotenv from 'dotenv'

var envFile = path.resolve(join(__dirname, "myenv.env"))
dotenv.config({path: envFile}) // Load the environment variables
console.log(`The connection URL is ${process.env.DATABASE_URL}`)
```

If you need variable expansion across environment files, you can additionally use [`dotenv-expand`](https://github.com/motdotla/dotenv-expand):

```ts
import * as dotenv from 'dotenv'
const dotenvExpand = require('dotenv-expand')

var envFile = path.resolve(join(__dirname, "myenv.env"))
var mySqlEnv = dotenv.config({path: envFile})
dotenvExpand.expand(mySqlEnv)
```
````